aws_lambda_powertools/utilities/streaming/compat.py

```diff
@@ -1,206 +1,3 @@
-                for line in lines[:-1]:
-                    yield line.splitlines(keepends)[0]
-                pending = lines[-1]
-            if pending:
-                yield pending.splitlines(keepends)[0]
-
-        def iter_chunks(self, chunk_size=_DEFAULT_CHUNK_SIZE):
-            """Return an iterator to yield chunks of chunk_size bytes from the raw
-            stream.
-            """
-            while True:
-                current_chunk = self.read(chunk_size)
-                if current_chunk == b"":
-                    break
-                yield current_chunk
-
-        def _verify_content_length(self):
-            # See: https://github.com/kennethreitz/requests/issues/1855
-            # Basically, our http library doesn't do this for us, so we have
-            # to do this ourself.
-            if self._content_length is not None and self._amount_read != int(self._content_length):
-                raise IncompleteReadError(
-                    actual_bytes=self._amount_read,
-                    expected_bytes=int(self._content_length),
-                )
-
-        def tell(self):
-            return self._raw_stream.tell()
-
-        def close(self):
-            """Close the underlying http response stream."""
-            self._raw_stream.close()
-
-    def convert_to_response_dict(http_response, operation_model):
-        """Convert an HTTP response object to a request dict.
-
-        This converts the requests library's HTTP response object to
-        a dictionary.
-
-        :type http_response: botocore.vendored.requests.model.Response
-        :param http_response: The HTTP response from an AWS service request.
-
-        :rtype: dict
-        :return: A response dictionary which will contain the following keys:
-            * headers (dict)
-            * status_code (int)
-            * body (string or file-like object)
-
-        """
-        response_dict = {
-            "headers": http_response.headers,
-            "status_code": http_response.status_code,
-            "context": {
-                "operation_name": operation_model.name,
-            },
-        }
-        if response_dict["status_code"] >= 300:
-            response_dict["body"] = http_response.content
-        elif operation_model.has_event_stream_output:
-            response_dict["body"] = http_response.raw
-        elif operation_model.has_streaming_output:
-            length = response_dict["headers"].get("content-length")
-            response_dict["body"] = PowertoolsStreamingBody(http_response.raw, length)
-        else:
-            response_dict["body"] = http_response.content
-        return response_dict
-
-    # monkey patch boto3
-    endpoint.convert_to_response_dict = convert_to_response_dict
-else:
-    from botocore.response import StreamingBody
-
-    # Expose PowertoolsStreamingBody as StreamingBody
-    vars()["PowertoolsStreamingBody"] = StreamingBody
+PowertoolsStreamingBody = StreamingBody
```

Comment:
```suggestion
# aliasing as 3.7 is no longer supported but unsure if anyone took a dependency on it (hyrum's law)
PowertoolsStreamingBody = StreamingBody
```

Reply:
a comment so we skip git blame if one ever needs to know what's this; later can be converted to `_compay.py` in v3.